docs/content/reference/dataframes.md

```diff
@@ -3,4 +3,43 @@ title: Dataframes
 order: 300
 ---
 
-Incoming.
+Rerun, at its core, is a database. As such, you can always get your data back in the form of tables (also known as dataframes, or records, or batches...).
+
+This can be achieved in three different ways, depending on your needs:
+* using the dataframe API, currently available in [Python](https://ref.rerun.io/docs/python/stable/common/dataframe/) and [Rust](https://docs.rs/crate/rerun/latest),
```

Comment:
We don't have the final docs.rs links yet, because docs.rs is timing out when processing 0.19.